documentation/todo-app/src/components/todoItems.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+export default class TodoItems extends Component {
+    render() {
+        const { description, deadline } = this.props;
+        
+        return (
+            <div> 
+                <tbody>
```

Comment:
Why are you using `tbody` and `tr`, etc, here? First, if you want to use these tags, they need to be inside a `table` tag. Second, there's no reason to use them! HTML tables are pretty complicated/annoying things. You can definitely achieve everything you want with simple divs.